### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product> Issues with haaretz.com

### DISCUSSION THREAD

**Discussion by Maxxxxxx** [community/community#130525] [Programming Help]
### Select Topic Area

Bug

### Body

Haaretz.com

I had to disable Bypass Paywall because of repeated errors: Details: cache-bur-kbur8200177-BUR 1719788200 1064043282
With it disabled, I can view the website but cannot read any articles beyond the headlines.
Never received this error previously and it took a while to figure it out.
Thank you. 
(Disabled UBlock also, because I get a popup.)

**Comment by LiteBrite82** [comment]
Hi @Maxxxxxx ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
